tests/ballerina-integration-test/src/test/resources/auth/authservices/07_auth_test_without_annotations.bal

```diff
@@ -0,0 +1,14 @@
+import ballerina/http;
```

Comment:
Shall we add the license header?

Reply:
Did not know we have started doing so for test source files. Will add.

Reply:
Added headers. Thanks!